src/pages/AzurePreview.js

```diff
@@ -0,0 +1,115 @@
+import { useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { ButtonOutline, ButtonPrimary, Link } from 'src/components/common'
+import { ReactComponent as AzureLogo } from 'src/images/azure.svg'
```

Comment:
I understand this is the old logo -- @lstrano-broad is reaching out to MSFT for an updated one.